src/components/License/License.js

```diff
@@ -168,6 +170,17 @@ class License extends React.Component {
             <LicenseAmendments {...this.getSectionProps('licenseAmendments')} />
             <SupplementaryDocs {...this.getSectionProps('licenseSupplement')} />
             <LicenseAgreements {...this.getSectionProps('licenseAgreements')} />
+            <NotesSmartAccordion
+              domainName="licenses"
+              entityName={data.license.name}
+              entityType="license"
+              entityId={data.license.id}
```

Comment:
Currently the Edit notes pane is crashing because the _NotesSmartAccordion_ needs **entityId** to always be defined when rendering it, and in this case here on initial load, it’s not being passed in because _data.license.id_ is still undefined at that time. So you would have to find a way to either defer the rendering until _data.license.id_ exists and is not undefined, or you would have to grab the **id** from the params passed via the _match_ prop like how we currently have for the same in _ui-agreements_